app/src/main/res/layout-land/profile_list_profile_view.xml

```diff
@@ -14,51 +14,53 @@
 
   <androidx.constraintlayout.widget.ConstraintLayout
     android:layout_width="match_parent"
-    android:layout_height="100dp">
+    android:layout_height="wrap_content"
+    android:background="@drawable/help_item_background_border">
 
     <de.hdodenhof.circleimageview.CircleImageView
       android:id="@+id/profile_list_image"
-      android:layout_width="68dp"
-      android:layout_height="68dp"
+      android:layout_width="40dp"
+      android:layout_height="40dp"
       android:layout_marginStart="36dp"
+      android:layout_marginTop="18dp"
+      android:layout_marginBottom="18dp"
+      android:background="@drawable/profile_image_shadow"
       app:layout_constraintBottom_toBottomOf="parent"
       app:layout_constraintStart_toStartOf="parent"
       app:layout_constraintTop_toTopOf="parent"
       profile:src="@{profile.avatar}" />
 
-    <LinearLayout
-      android:layout_width="wrap_content"
+    <TextView
+      android:id="@+id/profile_list_name"
+      android:layout_width="0dp"
       android:layout_height="wrap_content"
-      android:layout_marginStart="16dp"
-      android:orientation="vertical"
+      android:layout_marginStart="20dp"
+      android:layout_marginEnd="20dp"
+      android:ellipsize="end"
+      android:fontFamily="sans-serif"
+      android:maxLines="1"
+      android:text="@{profile.name}"
+      android:textColor="@color/black_87"
```

Comment:
Change this color to oppiaPrimaryTextDark.

Reply:
Done.